src/Database/Attach/File.php

```diff
@@ -976,6 +977,6 @@ protected function getPartitionDirectory()
      */
     protected function getLocalRootPath()
     {
-        return storage_path().'/app';
+        return Config::get('filesystems.disks.local.root');
```

Comment:
@pikanji This change is not necessary - the `getLocalRootPath` method is *only* called when the storage in use is "local".